### ISSUE TITLE
bug: dropdown on hover cannot open with keyboard only

### ISSUE THREAD

**Issue by TreyRuffy** [saadeghi/daisyui#1825]
### What version of daisyUI are you using?

2.51.6

### Describe your issue

The dropdown on hover component is unable to open using keyboard-only. 

Steps to reproduce:
1. Open https://daisyui.com/components/dropdown/#dropdown-on-hover
2. Press `tab` until the button looks like this image

![image](https://github.com/saadeghi/daisyui/assets/17056969/6977e9b9-5787-4820-b8ec-eda62e6e03c4)
3. Click space to open - will not open

### What browsers are you seeing the problem on?

Chrome, Firefox, Edge, Other

**Comment by saadeghi** [comment]
This is going to be fixed in version `3.0` update in 2 weeks (with many more bug fixes)  
But for now you can add this CSS as a temporary fix until version 3.0 gets ready:  
```
.dropdown.dropdown-hover:focus-within .dropdown-content {
  visibility: visible;
  opacity: 1;
}
```
Example: https://play.tailwindcss.com/nQ4MUo6eJX?file=css
